extra-kernel/linux-kernel/autobuild/amd64/config

```diff
@@ -11282,10 +11379,9 @@ CONFIG_DEBUG_INFO=y
 CONFIG_DEBUG_INFO_COMPRESSED=y
 # CONFIG_DEBUG_INFO_SPLIT is not set
 # CONFIG_DEBUG_INFO_DWARF_TOOLCHAIN_DEFAULT is not set
-CONFIG_DEBUG_INFO_DWARF4=y
-CONFIG_DEBUG_INFO_BTF=y
-CONFIG_PAHOLE_HAS_SPLIT_BTF=y
-CONFIG_DEBUG_INFO_BTF_MODULES=y
+# CONFIG_DEBUG_INFO_DWARF4 is not set
+CONFIG_DEBUG_INFO_DWARF5=y
```

Comment:
This option disables BTF info generation and may cripple some bpf functionality.